custom_components/tuya_local/devices/jiahong_et44w_thermostat.yaml

```diff
@@ -0,0 +1,194 @@
+name: Jiahong ET-44W thermostat
+entities:
+  - entity: climate
+    dps:
+      - id: 1
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: "off"
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: "smart"
+                value: auto
+              - dps_val: "hold"
+                value: heat
+              - dps_val: "holiday"
+                value: cool
+      - id: 16
+        type: integer
+        name: temperature
+        range:
+          min: 50
+          max: 400
+        mapping:
+          - scale: 10
+            step: 5
+            constraint: temperature_unit
+            conditions:
+              - dps_val: true
+                step: 30
+                range:
+                  min: 120
+                  max: 750
+      - id: 2
+        type: string
+        name: mode
+        hidden: true
+      - id: 24
+        type: integer
+        name: current_temperature
+        mapping:
+          - scale: 10
+      - id: 107
+        name: temperature_unit
+        type: boolean
+        mapping:
+          - dps_val: false
+            value: C
+          - dps_val: true
+            value: F
+      - id: 111
+        type: string
+        name: sensor
+        hidden: true
+      - id: 3
+        type: string
+        name: hvac_action
+        mapping:
+          - dps_val: "heating"
+            value: heating
+          - dps_val: "warming"
+            value: idle
+      - id: 104
+        type: integer
+        name: unknown_104
+      - id: 109
+        type: integer
+        name: holiday_days
+      - id: 112
+        type: integer
+        name: unknown_112
+      - id: 113
+        type: integer
+        name: unknown_113
+  - entity: lock
+    name: Screen lock
```

Comment:
Use `translation_key: child_lock` is better I think, then icon mappings can be removed.